packages/react-internal/src/components/SpinButton/SpinButton.types.ts

```diff
@@ -297,7 +272,7 @@ export interface ISpinButtonStyles {
   /**
    * Style for the icon when the control is disabled.
    */
-  iconDisabled: IStyle;
+  // iconDisabled: IStyle;
```

Comment:
Can you get rid of these commented props?

Reply:
Never mind I fixed this. Also updated the release notes about the removed props and added an example of what to do instead.

Reply:
Thank you!